R/get_existing_data_for_tests.R

```diff
@@ -31,6 +31,7 @@ get_existing_data_for_tests <- function(new_data, file_version = "episode") {
   if (file_version == "episode") {
     variable_names <- c(
       "anon_chi",
+      "cost_total_net_inc_dnas",
```

Comment:
```suggestion
      "cost_total_net",
```

Test - showing Tobi how to review a PR